kubecost-single-account-permissions.yaml

```diff
@@ -5,10 +5,10 @@ Description: Kubecost Masterpayer Account Template
 
 Parameters:
   AthenaCURBucket:
-    Description: The bucket where the CUR is sent from the “Setting up the CUR” step.
+    Description: The bucket where the CUR is sent from the "Setting up the CUR" step. This is in the AWS console>Billing Dashboard>Cost & usage reports>S3 Bucket for the daily report. https://docs.kubecost.com/install-and-configure/install/cloud-integration/aws-cloud-integrations#step-1-setting-up-the-cur
     Type: String
   SpotDataFeedBucketName:
-    Description: Optional. The bucket where the spot data feed is sent from the “Setting up the Spot Data feed” step.
+    Description: Optional- Kubecost uses actual billing for all cloud resources regardless of the spot feed being available. With the spot feed, near-term (48h) reports will use the spot rates. The bucket where the spot data feed is sent from the "Spot Data feed integration" step. This can be done at any time. https://docs.kubecost.com/install-and-configure/install/cloud-integration/aws-cloud-integrations/aws-spot-instances
```

Comment:
All for suggestions on this wording, it is a bit wordy.

Reply:
Why don't we link to docs?
https://docs.kubecost.com/install-and-configure/install/cloud-integration/aws-cloud-integrations/aws-spot-instances

Reply:
it is there, at the end.

Are you thinking- just link to docs without more description?